dex/router/src/factory.rs

```diff
@@ -83,35 +84,23 @@ pub trait FactoryModule {
                 self.blockchain().get_block_nonce(),
             ),
         );
+
         new_address
     }
 
-    fn upgrade_pair(
-        &self,
-        pair_address: ManagedAddress,
-        first_token_id: &TokenIdentifier,
-        second_token_id: &TokenIdentifier,
-        owner: &ManagedAddress,
-        _initial_liquidity_adder: &ManagedAddress,
-        total_fee_percent: u64,
-        special_fee_percent: u64,
-    ) {
-        self.pair_contract_deploy_proxy()
-            .contract(pair_address)
-            .init(
-                first_token_id,
-                second_token_id,
-                self.blockchain().get_sc_address(),
-                owner,
-                total_fee_percent,
-                special_fee_percent,
-                ManagedAddress::zero(),
-                MultiValueEncoded::new(),
-            )
-            .upgrade_from_source(
-                &self.pair_template_address().get(),
-                CodeMetadata::UPGRADEABLE | CodeMetadata::READABLE | CodeMetadata::PAYABLE_BY_SC,
-            );
+    fn upgrade_pair(&self, pair_address: ManagedAddress) {
```

Comment:
Why is this change needed? And in the claim developer rewards PR?

Reply:
It won't work with the current version of pair, it has no arguments for the upgrade function.